packages/dagu_car/src/kinematics_node.py

```diff
@@ -36,7 +37,7 @@ class KinematicsNode(DTROS):
         ~trim (:obj:`float`): trimming factor that is typically used to offset differences in the
             behaviour of the left and right motors, it is recommended to use a value that results in
             the robot moving in a straight line when forward command is given, default is 0.0
-        ~baseline (:obj:`float`): the distance between the two wheels of the robot, default is 0.1
+        ~wheel_distance (:obj:`float`): the distance between the two wheels of the robot, default is 0.1
```

Comment:
Can you also add  `turning_rad_lim` and `min_turn_rad`?

Reply:
Also in a number of places bellow you've added `wheel_distance`  but not  `turning_rad_lim` and `min_turn_rad`. Is there a reason for that?

Reply:
@AleksandarPetrov  @surirohit I am not sure if we want all of the parameters to be part of the kinematics calibration (i.e. the turning radius). I reverted the `wheel_distance` to `baseline` so it doesn't conflict with pre-existing calibration files. What do you think?

Reply:
Also because now we are reading only a part of them (`["gain", "trim", "wheel_distance", "k", "radius", "limit"]`) from the calibration file, while the others are disregarded i.e. `baseline`, `v_max`, `calibration_time`)

Reply:
@gibernas, this is a good question... not sure